### PR TITLE
Use navigator.languages as first choice as that is what Chrome supports

### PR DESCRIPTION
**Pull request by wytrych**
To fix bug in Chrome

### CODE REVIEW

**Comment by wytrych** [comment]
@incuna/js Please review

File: app/scripts/src/storage.js [hunk 1]
```diff
@@ -116,9 +116,17 @@ module.factory('languageStorage', [
             return navigatorLanguage.replace(/[-_].*/, '');
         };
 
+        const getUserLanguage = function () {
+            if ($window.navigator.languages) {
+                return $window.navigator.languages[0];
+            }
+
+            return $window.navigator.language || $window.navigator.userLanguage;
```
Comment (henrahmagix) [inline app/scripts/src/storage.js:130]: Can you comment to explain the connection to specific browsers? I.e. for `languages`
```js
// Chrome uses navigator.languages <for these reasons...>.
// Newer browsers will likely use this as it's the latest api.
```
and for `language`
```js
// Most browsers will use this.
```
and for `userLanguage`
```js
// Most IE browsers will use this.
```

In this case, I don't think "clean code" will provide the same explanation, so comments are good =)

**Review by henrahmagix — APPROVED** [review]

File: app/scripts/src/storage.js (hunk 1 above)
Comment (wytrych) [reply app/scripts/src/storage.js:130]: Done

**Review by henrahmagix — APPROVED** [review]
Top

**Pull request merged**